File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/comum/validators/rules/DefaultValidRule.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.comum.validators.rules;

import br.jus.trt8.pjecalc.base.comum.validadores.ValidatorContext;
import br.jus.trt8.pjecalc.negocio.comum.validators.ValidRule;
import br.jus.trt8.pjecalc.negocio.comum.validators.ValidValueValidator;
import br.jus.trt8.pjecalc.negocio.constantes.Mensagens;

public class DefaultValidRule
implements ValidRule {
    @Override
    public Mensagens getMessage() {
        return Mensagens.MSG0004;
    }

    @Override
    public boolean isValid(ValidValueValidator validator, ValidatorContext context) {
        return false;
    }
}

File: src/lib/pjecalc/core/comum/validators/rules/default-valid-rule.ts
/**
 * PJe-Calc v2.15.1 — DefaultValidRule
 * Porte de: br.jus.trt8.pjecalc.negocio.comum.validators.rules.DefaultValidRule
 */
import { Mensagens } from '../../mensagens';
import type { ValidRule } from '../valid-rule';
import type { ValidValueValidator } from '../valid-value-validator';
import type { ValidatorContext } from '../validator-context';

export class DefaultValidRule implements ValidRule {
  getMessage(): Mensagens { return Mensagens.MSG0004; }
  isValid(_validator: ValidValueValidator, _context: ValidatorContext): boolean {
    return false;
  }
}
